ForJarik

Merchanskyi opened the pull request:
Up

Merchanskyi committed:
dlya Jarika

Merchanskyi committed:
dlyaJarika

Merchanskyi committed:
forJarik

Merchanskyi committed:
up

YaroslavKuchuk commented:
1. Все твои коммиты должны быть информативными. Не просто "dlyaJarika", а, например, "Стартовый коммит", "Структура классов", "Поток для добавления записей", и т.д.

YaroslavKuchuk commented on ModuleThreeApp/LibraryApp/DCAction.cs:
```diff
@@ -0,0 +1,9 @@
+{
+    public enum DCAction
+    {
+        GetServers = 1,
```
Константы по порядку - 0, 1, 2

Merchanskyi replied:
Понял

Merchanskyi replied:
И вверху по поводу dlyaYarika так же понял=) буду писать что изменилось

YaroslavKuchuk commented on ModuleThreeApp/LibraryApp/DataCenter.cs:
```diff
@@ -0,0 +1,25 @@
+    {
+        public string Name { get; set; }
+
+        public List<Server> Servers { get; set; } = new List<Server>();
```
Сможем сделать ее кастомной (отдельной) коллекцией?

Merchanskyi replied:
Сделаю

YaroslavKuchuk commented on ModuleThreeApp/ModuleThreeApp/LogicAction.cs:
```diff
@@ -0,0 +1,167 @@
+                        foreach (var dataCenter in DataCenters)
+                        {
+                            Console.WriteLine(dataCenter.ToString());
+                            //OutputInfo(dataCenter.Servers);
```
Старайся не оставлять комментарии в коде.

Merchanskyi replied:
Понял, уберу

YaroslavKuchuk commented on ModuleThreeApp/ModuleThreeApp/LogicAction.cs:
```diff
@@ -0,0 +1,167 @@
+
+                        //Второй вариант
+                        /*
+                        for (int i = 0; i < DataCenters.Count; i++)
```
См. выше

YaroslavKuchuk commented on ModuleThreeApp/LibraryApp/Server.cs:
```diff
@@ -0,0 +1,19 @@
+    {
+        public string Name { get; set; }
+
+        public string AdditionalPower { get; set; }
```
Может его стоит сделать не строкой?

Merchanskyi replied:
AdditionalPower?

yaku1202 replied:
да. По-моему, это точно не строка.

YaroslavKuchuk commented on ModuleThreeApp/LibraryApp/Server.cs:
```diff
@@ -0,0 +1,19 @@
+
+        public string AdditionalPower { get; set; }
+
+        public int BaseType { get; set; }
```
Как насчет enum?

Merchanskyi replied:
Тут не совсем понял, AdditionalPower и BaseType сделать нужно enum'ами?

yaku1202 replied:
Не, BaseType.

YaroslavKuchuk commented:
Тоже самое касается бранчей.

YaroslavKuchuk reviewed (changes requested):
Пожалуйста, поменяй типы (и остальные комментарии). И обсудим в понедельник. Продлжай также. Хорошая работа! )

Merchanskyi committed:
1) Delete second List<DataCenter>.
2) Added the option when the file is not created or does not exist.
3) Added checking when the file is empty.
4) Remove 'BaseType'.
